conda/repodata.py

```diff
@@ -0,0 +1,99 @@
+# (c) Continuum Analytics, Inc. / http://continuum.io
```

Comment:
Also, a good indicator of what is API should be "does it have a very good docstring"?